indexer-app/src/cmr/indexer/data/index_set_elasticsearch.clj

```diff
@@ -34,7 +34,7 @@
         ;; will know which one caused the issue by seing which index was the last
         ;; to start
         (when (= "dev" (cmr.common-app.config/release-version))
-          (println "Here in create-index function creating" index-name)
+          (println "This instance of CMR will publish to Elastic Index:" index-name)
```

Comment:
Just a thought, do we want to make this an (info log?

Reply:
I'm going to forget, I'll do this now. Erich and I had the exact same conversation.